package.json

```diff
@@ -15,7 +15,8 @@
     "test": "npm run lint && mocha --compilers js:babel-register tests --recursive",
     "semantic-release": "semantic-release",
     "commitmsg": "npm run test && commitlint -e $GIT_PARAMS",
-    "build": "rm -fr ./dist && babel ./src --out-dir ./dist --copy-files"
+    "build": "rm -fr ./dist && babel ./src --out-dir ./dist --copy-files",
```

Comment:
Looks like this is referencing `src`, but that folder doesn't exist - it's supposed to be `lib`.

Reply:
Thanks https://github.com/DianaSuvorova/eslint-plugin-react-redux/pull/82